pkg/cloud/amazon/ec2/ec2.go

```diff
@@ -0,0 +1,99 @@
+package ec2
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/client"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/jenkins-x/jx/pkg/cluster"
+	"github.com/jenkins-x/jx/pkg/log"
+
+	"github.com/aws/aws-sdk-go/aws"
+
+	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
+
+	"github.com/pkg/errors"
+)
+
+// EksClusterOptions contains some functions to interact with EKS
+type ec2Options struct {
```

Comment:
GoDocs. Also, I am wondering about the name itself `ec2Options`, in particular 'option'. I feel we use 'option' a bit too much in our code base ;-) `ec2Connector`, `ec2Handler`?

Reply:
Or if more going the Go way `ec2er`

Reply:
I like idiomatic Go, you think `ec2er` is more idiomatic? :)

Reply:
I think so